Jenkinsfile

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/groovy
+// groovylint-disable DuplicateMapLiteral, DuplicateNumberLiteral
+// groovylint-disable DuplicateStringLiteral, NestedBlockDepth
+
```

Comment:
I will file a ticket, however with out these suppressions, when the groovylint plugin is installed, the editor display for visual studio code for the Jenkinsfile is not readable.   Almost all the lines are flagged because the Jenkinsfile is a special case of groovy.   So until I get this PR passing, I am going to have to leave these suppressions in.

Reply:
These tools are great, I've recently changed my editor and am also suffering from having many hundreds of errors shown in files that we use/modify every day.  It's a good thing to work on and resolve those issues.

The issues shown are the tools way of highlighting to us potential issues so if we broadly suppress them then we're closing our eyes and ears to advice that would help us improve our codebase.